.github/workflows/core.yml

```diff
@@ -388,7 +388,7 @@ jobs:
       - run: chmod +x core/build/unix/trezor-emu-core*
       - uses: ./.github/actions/environment
       - run: nix-shell --run "tests/download_emulators.sh"
-      - run: nix-shell --run "poetry run pytest tests/upgrade_tests"
+      - run: nix-shell --run "poetry run pytest --durations 50 tests/upgrade_tests"
```

Comment:
Please add the option to `TESTOPTS` env var if possible.

Reply:
8e9ae6460d

Reply:
8e9ae6460d